apps/ledger-live-mobile/src/screens/Settings/Debug/Debugging/CameraPermissions.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import { Button, Flex, Switch, Text } from "@ledgerhq/native-ui";
+import { Camera } from "expo-camera";
+import { useIsFocused } from "@react-navigation/native";
+import { CameraType } from "expo-camera/build/Camera.types";
+import RequiresCameraPermissions from "../../../../components/RequiresCameraPermissions";
+import CameraPermissionContext from "../../../../components/RequiresCameraPermissions/CameraPermissionContext";
+
+const CameraPermissions: React.FC<{}> = () => {
```

Comment:
TS is complaning that `{}` means any non-nullish value.

Either remove the `React.FC<{}>` type all together and let TS infers the type. Or use `React.FC<Record<string, never>>` for the type

Reply:
nice nice thank you!